tests/refs/dup.c

```diff
@@ -0,0 +1,40 @@
+#include "clar_libgit2.h"
+#include "refs.h"
+
+static git_repository *g_repo;
+
+void test_refs_dup__initialize(void)
+{
+	g_repo = cl_git_sandbox_init("testrepo.git");
+}
+
+void test_refs_dup__cleanup(void)
+{
+	cl_git_sandbox_cleanup();
+}
+
+void test_refs_dup(void)
```

Comment:
This test needs a proper name - `test_refs_dup` is the prefix for the `refs::dup` tests.  This test needs to be `test_refs_dup__direct` (for example).  (Clar is quite particular about naming.)

(You can run just the `refs::dup` tests to see that it only runs the one.)